src/pages/edit-profile/EditProfile.tsx

```diff
@@ -87,6 +88,10 @@ const EditProfile = () => {
       )
     }
     void fetchData()
+
+    return () => {
+      void fetchData()
+    }
```

Comment:
Why `fetchData` is using as cleanup function?

Reply:
We have a logic where the photo updates immediately when the user selects a new one (either by clicking the "Select" or "Remove Photo" button), even before clicking the "Update" button. However, when the user leaves the edit page, the photo that hasn’t been updated (saved to the server) is still shown as their profile icon in the top-right corner.
To address this, before the component is unmounted and the user leaves the edit profile page, I am refetching the data from the server to ensure that only the correctly updated photo is displayed.

Reply:
Oh, I got it
It's very thoughtfully, really)